modules/planning/common/frame.h

```diff
@@ -121,11 +121,11 @@ class Frame {
       const double planning_start_time,
       prediction::PredictionObstacles *prediction_obstacles);
 
-  void set_last_planned_trajectory(ADCTrajectory last_planned_trajectory) {
+  void set_current_frame_planned_trajectory(ADCTrajectory last_planned_trajectory) {
```

Comment:
Should be either "const ADCTrajectory& last_planned_trajectory" or "ADCTrajectory&& last_planned_trajectory", or else it does several copies.